plugins/public/cinq-collector-aws/cinq_collector_aws/account.py

```diff
@@ -52,13 +62,86 @@ def run(self):
             if self.route53_collection_enabled:
                 self.update_route53()
 
+            if self.rds_collection_enabled:
+                self.update_rds_databases()
+
         except Exception as ex:
             self.log.exception(ex)
             raise
 
         finally:
             del self.session
 
+    @retry
+    def update_rds_databases(self):
+        """Update list of RDS Databases for the account / region
+
+        Returns:
+            `None`
+        """
+        self.log.debug('Updating RDS Databases for {}'.format(
+            self.account.account_name
+        ))
+        # All RDS resources are polled via a Lambda collector in a central account
+        rds_session = get_aws_session(self.rds_collector_account)
+        # Existing RDS resources come from database
+        existing_rds_dbs = RDS.get_all(self.account)
+
+        try:
+            # Special session pinned to a single account for Lambda invocation so we
+            # don't have to manage lambdas in every account & region
+            lambda_client = rds_session.client('lambda', region_name=self.rds_collector_region)
+
+            # The AWS Config Lambda will collect all the non-compliant resources for all regions
+            # within the account
+            input_payload = {"AccountId": self.account.account_id,
+                             "RoleName": self.rds_role
+                             }
+
+            rds_dbs = lambda_client.invoke(FunctionName=self.rds_function_name, InvocationType='Event',
+                                           PayLoad=input_payload
+                                           )
+
+            for db in rds_dbs.values:
+                tags = {t['Key']: t['Value'] for t in db.tags or {}}
+                properties = {
+                    'metrics': {db.metrics},
```

Comment:
Unnecessary brackets?

Reply:
yeah, metrics is a dict but it would already be defined as such, thanks.